harness/determined/common/experimental/determined.py

```diff
@@ -419,39 +419,47 @@ def remove_oauth_client(self, client_id: str) -> None:
     def stream_trials_metrics(
         self, trial_ids: List[int], group: str
     ) -> Iterable[metrics.TrialMetrics]:
-        """
-        Streams metrics for one or more trials sorted by
-        trial_id, trial_run_id and steps_completed.
+        """Get a stream of metrics for this trial.
 
         Arguments:
-            trial_ids: List of trial IDs to get metrics for.
+            trial_ids: The trial IDs to stream metrics for.
+            group: The metric group to stream.  Valid values are "validation" and "training".
```

Comment:
Technically, the valid values could be anything, it's just that the "validation" and "training" are what we originally had, and now with generic metrics the group could be anything the user specifies